src/Cake.Scripting/Cake.Scripting.csproj

```
@@ -15,6 +15,7 @@
     <PackageReference Include="Cake.Core" Version="1.3.0" />
     <PackageReference Include="Cake.Common" Version="1.3.0" />
     <PackageReference Include="Mono.Cecil" Version="0.10.1" />
+	<PackageReference Include="Basic.Reference.Assemblies.Net60" Version="1.2.4" />
```

Replace tabs with spaces:
```suggestion
    <PackageReference Include="Basic.Reference.Assemblies.Net60" Version="1.2.4" />
```

---

I haven't checked this out, so maybe check your other changes for tabs?